rocAL/rocAL/source/parameters/parameter_rocal_crop.cpp

```diff
@@ -27,42 +27,41 @@ THE SOFTWARE.
 #include "parameter_rocal_crop.h"
 #include "commons.h"
 
-void RocalCropParam::set_crop_height_factor(Parameter<float>* crop_h_factor)
-{
+void RocalCropParam::set_crop_height_factor(Parameter<float>* crop_h_factor) {
     if(!crop_h_factor)
         return ;
     ParameterFactory::instance()->destroy_param(crop_height_factor);
     crop_height_factor = crop_h_factor;
 }
 
-void RocalCropParam::set_crop_width_factor(Parameter<float>* crop_w_factor)
-{
+void RocalCropParam::set_crop_width_factor(Parameter<float>* crop_w_factor) {
     if(!crop_w_factor)
         return ;
     ParameterFactory::instance()->destroy_param(crop_width_factor);
     crop_width_factor = crop_w_factor;
 }
 
-void RocalCropParam::update_array()
-{
+void RocalCropParam::update_array() {
     fill_crop_dims();
     update_crop_array();
 }
 
-void RocalCropParam::fill_crop_dims()
-{
-    for(uint img_idx =0; img_idx < batch_size; img_idx++)
-    {
-        if(!(_random))
-        {
+void RocalCropParam::fill_crop_dims() {
+    for(uint img_idx =0; img_idx < batch_size; img_idx++) {
+        if(!(_random)) {
             // Evaluating user given crop
-            (crop_w > in_width[img_idx]) ? (cropw_arr_val[img_idx] = in_width[img_idx]) : (cropw_arr_val[img_idx] = crop_w);
-            (crop_h > in_height[img_idx]) ? (croph_arr_val[img_idx] = in_height[img_idx]) : (croph_arr_val[img_idx] = crop_h);
-            (x1 >= in_width[img_idx]) ? (x1_arr_val[img_idx] = 0) : (x1_arr_val[img_idx] = x1);
-            (y1 >= in_height[img_idx]) ? (y1_arr_val[img_idx] = 0) : (y1_arr_val[img_idx] = y1);
+            cropw_arr_val[img_idx] = (crop_w <= in_width[img_idx] && crop_w > 0) ? crop_w : in_width[img_idx];
```

Comment:
Crop can start from 0 ryt?

Reply:
cropw_arr_val - This is the width value which cannot be "0"